copter/source/docs/traditional-helicopter-parameter-list.rst

```diff
@@ -34,11 +34,11 @@ These set the cyclic max, allow bench testing of blade angle setup, and setting
 Single Swashplate/Collective Setup
 ==================================
 
-These setup the configuration of the swashplate and collective travel.
+These set up the configuration of the swashplate and collective travel.
 
 - :ref:`H_SW_TYPE<H_SW_TYPE__AP_MotorsHeli_Single>`
-- :ref:`H_SW_H3_ENABLE<H_SW_H3_ENABLE__AP_MotorsHeli_Single>`  Never manually set this parameter!
-- ``H_COLYAW<H_COLYAW`` This parameter will be eliminated in versions after 4.3
+- :ref:`H_SW_H3_ENABLE<H_SW_H3_ENABLE__AP_MotorsHeli_Single>` Never manually set this parameter!
+- :ref:`H_COL2YAW<H_COLYAW>` This parameter will be eliminated in versions after 4.3
```

Comment:
this is incorrect....please revert

Reply:
Where should the new parameter be referenced then?

Reply:
it will be when I get to it on the 4.4 list to add to the future wiki changes